src/server/game/Battlegrounds/ArenaTeam.h

```diff
@@ -188,15 +188,15 @@ class TC_GAME_API ArenaTeam
 
     protected:
 
-        uint32      TeamId;
-        uint8       Type;
+        uint32 TeamId;
+        uint8 Type;
         std::string TeamName;
-        ObjectGuid  CaptainGuid;
+        ObjectGuid CaptainGuid;
 
         uint32 BackgroundColor; // ARGB format
-        uint8  EmblemStyle;     // icon id
+        uint8 EmblemStyle;      // icon id
         uint32 EmblemColor;     // ARGB format
-        uint8  BorderStyle;     // border image id
+        uint8 BorderStyle;      // border image id
         uint32 BorderColor;     // ARGB format
 
         MemberList     Members;
```

Comment:
please remove whitespace from all the members of this class, starting from https://github.com/TrinityCore/TrinityCore/pull/23519/files#diff-d7bafcb23b1ab0c2d6dffc8cd7a74d87R116